docs/en/reference/working-with-associations.rst

```diff
@@ -488,30 +471,52 @@ and the "remove" operation.
         //...
     }
 
-Since ``cascade: persist`` is configured for the ``User#commentsAuthored``
-association, you can now create a user and persist their comments as follows:
+...you can now create a user and an associated comment like this:
 
 .. code-block:: php
 
     <?php
     $user = new User();
-    $myFirstComment = new Comment();
-    $user->addComment($myFirstComment);
-    $myFirstComment->setUser($user);
+    $user->comment('Lorem ipsum', new DateTime());
```

Comment:
The example you're replacing sets the `user` property in the comment, but the `comment` method written above does not do that. You should add that to the example.